modules/atomic-widgets/elements/atomic-tabs/atomic-tabs.php

```diff
@@ -173,7 +173,7 @@ protected function add_render_attributes() {
 		];
 
 		if ( ! empty( $settings['default-active-tab'] ) ) {
-			$attributes['data-active-tab'] = esc_attr( $settings['default-active-tab'] );
+			$attributes['data-e-settings'] = json_encode( [ 'default-active-tab' => $settings['default-active-tab'] ] );
```

Comment:
`$settings['default-active-tab']` should be escaped but for the context of js.

check if https://developer.wordpress.org/reference/functions/esc_js/ helps